docs/pages/reference/api.md

```
@@ -135,34 +126,4 @@ curl -X POST \
-```
-> Maps to `path/file.md`
-
-**Directory default**
-
-Directory path defaults to `index.md` (for human-readable navigation):
-
-```bash
-curl https://your-app.app.statespace.com/path/
-```
-> Maps to `path/index.md`
+</div>
```

markdownlint-cli2 (0.21.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)